src/Create.js

```diff
@@ -0,0 +1,33 @@
+import React, {useState, useEffect} from 'react';
+import Firestore from './services/firestore.js';
+import Board from './Board';
+
+function Create(props) {
+  const [grid, setGrid] = useState([]);
+
+  async function saveGrid(e) {
+    e.preventDefault();
+    let fields = e.target.elements;
+    let email = fields['email'].value;
+    let title = fields['title'].value;
+    const createdGrid = await Firestore.add.grid(email, title, grid);
```

Comment:
Not a problem right now, but later in the future, you and Bryan need to talk about passing the user information from the welcome/nav bar to this file. Then the user's id can be retrieved with user.id instead of manually entering the email into a text field.